src/main/proto/dictionary.proto

```diff
@@ -271,6 +271,7 @@ message Process {
 	string browser_uuid = 14;
 	string form_uuid = 15;
 	string workflow_uuid = 16;
+	bool is_has_parameters = 17;
```

Comment:
This is innecessary because already exist this definition: https://github.com/solop-develop/backend/blob/c5aa9b3563aeeccfe99551e13a4ac94f40c82f93/src/main/proto/dictionary.proto#L265

Reply:
change removed.